src/apps/lwaftr/binding_table.lua

```diff
@@ -143,10 +143,31 @@ local BindingTable = {}
 function BindingTable.new(psid_map, br_addresses, softwires)
    local ret = {
       psid_map = assert(psid_map),
-      br_addresses = assert(br_addresses),
-      softwires = assert(softwires)
+      softwires = assert(softwires),
+      br_addresses = cltable.new({key_type = ffi.typeof('uint8_t[16]')})
```

Comment:
I don't understand this part. My understanding is that after this change there won't be a `br_addresses` table, so why it's necessary to initialize it? I see that the uses of `br_addresses` were remove in the code too. Are the `add_br_address` and `remove_br_address` aimed to be used from outside?

Lastly, since `br_addresses` is initialized out of the softwires maybe is possible to remove the `br_addreses` argument. I understand that this argument contains the value of parsing the br_addresses field using the v1 binding table format or nil.